action.yml

```diff
@@ -28,5 +36,6 @@ runs:
     - name: Run Kani
       shell: bash
       run: | 
-        ${{ inputs.command }}
+        cd ${{ inputs.working-directory }}; \
```

Comment:
why `\`?

Reply:
also, do we even need this? I would expect github action to start from the working directory.

Reply:
s2n-quic does this, which we want to replicate in an ergonomic manner
https://github.com/aws/s2n-quic/blob/main/.github/workflows/ci.yml#L679

Reply:
it looks like the working directory only applies to `run` and not `uses`. :(

Reply:
I was just wondering if you could leverage the working-directory from github. Like here: https://github.com/aws/s2n-quic/blob/main/.github/workflows/ci.yml#L198

which I believe is the most ergonomic manner. :)